hazelcast/src/main/java/com/hazelcast/map/impl/MapDataSerializerHook.java

```diff
@@ -52,8 +53,9 @@
     public static final int QUERY_RESULT = 10;
     public static final int EVICT_BACKUP = 11;
     public static final int CONTAINS_KEY = 12;
+    public static final int KEY_ITERATOR = 13;
 
-    private static final int LEN = CONTAINS_KEY + 1;
+    private static final int LEN = KEY_ITERATOR + 1;
 
```

Comment:
It's just a convention which was used throughout the serializer hooks, I don't want to be the outlier in this case :) 

For JavaDocs, there will be a followup PR for this change which includes some more edge case tests. I plan to add JavaDocs on that PR.